OpenEdXMobile/res/layout-land/layout_course_dashboard_certificate.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/neutralWhite"
```

Comment:
Please use `neutralWhiteT` color

Reply:
In landscape, the color is set to `neutralWhite`.